copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -10,63 +10,54 @@ Autopilot Info
 .. image:: ../images/pixhackv5.jpg
     :target: ../_images/pixhackv5.jpg
 
-A autopilot with internally damped IMU's is highly recommended for 
-helicopters. Experience has shown the tuning, handling and stability 
-performance of your helicopter will be greatly improved over the first 
-generation Pixhawk.
+A autopilot with internally damped IMU's is highly recommended for helicopters. Experience has shown the tuning, handling and stability performance of your helicopter will be greatly improved over the first generation Pixhawk.
 
-Before you begin connecting the system it is recommended to review the docs for
-the autopilot you select.
+Before you begin connecting the system it is recommended to review the docs for the autopilot you select.
 
 Overview of servo, and RX connection
 ====================================
 
-The RC input for many ardupilot compatible autopilots is either PPM SUM
-(8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum
-satellite receivers.  If you have a receiver that outputs only PWM, you will need
-a PPM encoder to connect to your autopilot.
-
-The default RC input and SERVO output mapping is as follows:
-
-+--------------+-------------+
-| RC Channel   | SERVO output|
-+--------------+-------------+
-| 1            | Aileron     |
-+--------------+-------------+
-| 2            | Elevator    |
-+--------------+-------------+
-| 3            | Collective  |
-+--------------+-------------+
-| 4            | Rudder      |
-+--------------+-------------+
-| 5            | Flight Modes|
-+--------------+-------------+
-| 6            | Tuning      |
-+--------------+-------------+
-| 7            | Aux         |
-+--------------+-------------+
-| 8            | Throttle    |
-+--------------+-------------+
+The RC input for many ardupilot compatible autopilots is either PPM SUM (8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum satellite receivers.  For receivers that only output PWM, a PPM encoder is required to connect to the autopilot.
```

Comment:
might be nice to state that the minimum # RC inputs channels is (x....think its 6?) and outputs is (y...thinks its 5?) since we have many different kinds of autopilots and rc systems

Reply:
Minimum input channels is 8, min output is 6

Reply:
you mean you cant fly without tuning or aux? I know setup is going to be difficult without them, but you cant fly without them?
5 outputs are: 3 for swash/collective, rsc, yaw....what is the 6th? you have to have yaw mode/tuning? bear in mind I am coming from flybar days on helis...
just coming from the simplest point of view...

Reply:
Technically, in 4.0 I think a 6 channel RC radio will work. The RSC, or throttle, used to be hard-wired to channel 8 in heli.cpp. That has changed in 4.0.

I guess if a controller exists with only 5 outputs, it could probably be used. Is there any like that? Even the Pixracer has 6.

Reply:
with the new board alt configs there could be...I made a custom omnibusf4pro compile for a guy that changed one of the 6 pwms to get another uart output, that might be a standard alternate in the future...no telling what a future board might happen...but the real point is understanding what is or is not possible...I have an autonomous rover that only uses two channels on rc...I have to change modes with my phone gcs...just trying to understand what is or is not possible..